SourceX/dvlnet/frame_queue.cpp

```diff
@@ -48,7 +48,15 @@ bool frame_queue::packet_ready()
 		if(size() < sizeof(framesize_t))
 			return false;
 		auto szbuf = read(sizeof(framesize_t));
+#if defined(_WIN64) || (defined(__x86_64__) && !defined(__ILP32__)) ||	\
+	defined(__LP64__) || defined(_M_X64) || defined(_M_ARM64) ||	\
+	defined(__aarch64__) || defined(__ppc64__) ||			\
+	defined(__powerpc64__) || defined(_ARCH_PPC64)
+		std::memcpy(&nextsize, &szbuf[0], sizeof(framesize_t));
+		nextsize >> 32;
```

Comment:
Is `nextsize >> 32;` a nop? Since it doesn't store the result.

Reply:
Interesting. Yeah I had intended >= but actually left it in as a no-op, luckily. Changing it to >= caused player 1 to fail to even host the game. Thank you for refactoring and reviewing.